src/agent.spec.ts

```diff
@@ -5,88 +5,86 @@ import {
   createTransactionEvent,
   HandleTransaction,
   TransactionEvent,
-  
+
 } from "forta-agent";
 import { TestTransactionEvent } from "forta-agent-tools/lib/test";
-import provideHandleTransaction from "./agent";
+import { createAddress } from "forta-agent-tools";
 import { Interface } from "@ethersproject/abi";
-import { FORTA_BOT_REGISTRY,BOT_DEPLOYER_ADDRESS, NEW_AGENT_FUNCTION_SIGNATURE} from "./constants";
+import { FORTA_BOT_REGISTRY, BOT_DEPLOYER_ADDRESS, NEW_AGENT_FUNCTION_SIGNATURE, TEST_AGENT_FUNCTION_SIGNATURE } from "./constants";
```

Comment:
If you run the `prettier` script occasionally while you're building, it'll clean up your code. Run it, and it should clean up this line to be like the `forta-agent` import above.